### PR TITLE
[+] Command Generate - Add support for MySQL 8

### PR DESCRIPTION
**Pull request by rap2hpoutre**
- Tests works locally and remotely.
 - No need to downgrade password.
 - `mysql2` has been updated to latest MAJOR version (2) (https://github.com/sidorares/node-mysql2/blob/master/Changelog.md)

### CODE REVIEW

**Comment by rap2hpoutre** [comment]
[Database - Make Forest compatible with MySQL 8.0](https://trello.com/c/KGrkpuKT/128-database-make-forest-compatible-with-mysql-80)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 ## [Unreleased]
+### Added
+- Command Generate - Add support for MySQL 8.
+
 ### Fixed
 - Command Generate - Fix error on project generation with database contain table with number as name.
 
+### Changed
+- Tests - Use MySQL 8.0 as the max supported MySQL version.
+- Technical - Update `mysql2` dependency to 2 (latest MAJOR version).
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: Same question as in the `forest-express-sequelize` repository:
Is this a necessary change to support MySQL 8?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,16 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Command Generate - Add support for MySQL 8.
+
 ### Fixed
 - Command Generate - Fix error on project generation with database contain table with number as name.
 
+### Changed
```
Comment (arnaudbesnier) [inline CHANGELOG.md]: [Conventions]
The "Changed" section is always above "Fixed" one.

File: CHANGELOG.md (hunk 1 above)
Comment (rap2hpoutre) [reply CHANGELOG.md]: See: https://github.com/ForestAdmin/forest-express-sequelize/pull/373#discussion_r374221991

Comment (arnaudbesnier) [reply CHANGELOG.md]: Same response ;)
https://github.com/ForestAdmin/forest-express-sequelize/pull/373#discussion_r374246697

**Pull request merged**